dist/autocomplete.jquery.js

```diff
@@ -372,11 +372,12 @@
 	  $input = this.$node.find(_.className(this.cssClasses.prefix, this.cssClasses.input));
 	  $hint = this.$node.find(_.className(this.cssClasses.prefix, this.cssClasses.hint));
 
-	  if (o.dropdownMenuContainer) {
-	    DOM.element(o.dropdownMenuContainer)
-	      .css('position', 'relative') // ensure the container has a relative position
-	      .append($menu.css('top', '0')); // override the top: 100%
-	  }
+	  DOM.element('body').append($menu);
+	  window.$input = $input;
+	  $menu.
+	    css('top', $input.offset().top + $input.height()).
```

Comment:
Actually the autocomplete starts at the middle of the field:
![image](https://cloud.githubusercontent.com/assets/5570853/15649802/86147928-2674-11e6-97eb-1fd437f75271.png)
Replacing `$input.height()` by `$input.outerHeight()` fixes it.